### PR TITLE
Drop unused createdForResource and createdForResourceType labels

### PR DESCRIPTION
**Pull request by ahadas**

### CODE REVIEW

**Review by liranr23 — APPROVED** [review]

**Review by mnecas — COMMENTED** [review]
I don't think we can remove it. The UI plugin uses it in [1]. 

[1] https://github.com/kubev2v/forklift-console-plugin/blob/c4a5046d0eb3be930da0a322e44c391a2c0c205f/packages/legacy/src/client/helpers.ts#L228

**Comment by ahadas** [comment]
> I don't think we can remove it. The UI plugin uses it in [1].

also, we don't really drop it here - but remove the places that remain on the backend side. if the standalone UI set those label, it's not that great as it's confusing, but the standalone UI should be used anymore besides some exceptions

File: tests/suit/ovirt_test.go [hunk 1]
```diff
@@ -30,8 +30,6 @@ var _ = Describe("[level:component]Migration tests for oVirt provider", func() {
 		By("Create Secret from Definition")
 		s, err := utils.CreateSecretFromDefinition(f.K8sClient, utils.NewSecretDefinition(
 			map[string]string{
-				"createdForResource":     "ovirt-provider",
-				"createdForResourceType": "providers",
 				"createdForProviderType": "ovirt",
```
[the remark this answers is not in the document]
Comment (yaacov) [reply tests/suit/ovirt_test.go:33]: maybe an owner makes more sense ?

**Review by mnecas — APPROVED** [review]

**Pull request merged**